### PR TITLE
validity_period files were removed

### PR DESCRIPTION
**Pull request by gromeroar**
Validity period files removed from the project
## Description

All the validity period files and dependencies were removed from the project.
## Checklist:

- [x] I have added a [Signed-off-by](https://github.com/hyperledger/fabric/blob/master/CONTRIBUTING.md#legal-stuff)
- [X] Either no new documentation is required by this change
- [X] Either no new tests are required by this change
- [X] I have run [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports), [go vet](https://golang.org/cmd/vet/), and [golint](https://github.com/golang/lint). I have cleaned up all valid errors and warnings in code I have added or modified. These tools may generate false positives. Don't be worried about ignoring some errors or warnings. The goal is clean, consistent, and readable code.

Signed-off-by: gromeroar guillermo.romero@gmail.com / rologui@ar.ibm.com

### CODE REVIEW

**Comment by srderson** [comment]
Could you give a bit more background regarding this change? I'm out of context. Also, there is a merge conflict so this needs to be rebased.

**Comment by diegomasini** [comment]
@srderson this is old functionality not being used for the previous implementation of validity period without system chaincode support. This should be reimplemented using system chaincode and role based access control.

**Comment by JonathanLevi** [comment]
@srderson while you are at it... another look at this one please?
What @diegomasini said (the conf was explicitly disabled for a while too).

**Comment by JonathanLevi** [comment]
@srderson while you are at it... another look at this one please?
What @diegomasini said (the configuration was explicitly disabling these for a while too). Please see also: https://github.com/hyperledger/fabric/pull/1781/files#diff-201583256d45f53f52fffb9adfb11919L63

> We are disabling the validity period deployment for now, we shouldn't even allow it if it's enabled in the configuration
> 
> ## \-       allowDeployValidityPeriod := false

(No idea why the font is so large ;-))

File: core/ledger/genesis/genesis.go [hunk 1]
```diff
@@ -55,26 +55,11 @@ func MakeGenesis() error {
 
 		defer func() {
 			if !genesisBlockExists && makeGenesisError == nil {
-				genesisLogger.Infof("Adding %d system chaincodes to the genesis block.", len(genesisTransactions))
+				genesisLogger.Info("Adding %d system chaincodes to the genesis block.", len(genesisTransactions))
```
Comment (srderson) [inline core/ledger/genesis/genesis.go]: Why was this changed from `Infof` to `Info`? Same question for the other logging changes in this file.

**Comment by srderson** [comment]
@gromeroar There are a whole bunch of cases of the same problem (`Infof` -> `Info`, `Debugf` -> `Debug`, etc.) in `genesis.go`. You only fixed the one I pointed out directly. These need to be fixed before this PR can be accepted.

**Comment by gromeroar** [comment]
the log changes have been fixed, I've added the log with F of format postfix to all the info, error, debug, warning appenders which needs log text with parameters.

File: core/ledger/genesis/genesis.go [hunk 2]
```diff
@@ -95,21 +80,21 @@ func MakeGenesis() error {
 				chaincode := chaincodes[name]
 				chaincodeMap, chaincodeMapOK := chaincode.(map[interface{}]interface{})
 				if !chaincodeMapOK {
-					genesisLogger.Errorf("Invalid chaincode defined in genesis configuration: %s", chaincode)
+					genesisLogger.Error("Invalid chaincode defined in genesis configuration:", chaincode)
```
Comment (srderson) [inline core/ledger/genesis/genesis.go]: I don't understand why you would remove the `%s` in this case and change `Errorf` to `Error`. Am I missing something?

Comment (srderson) [inline core/ledger/genesis/genesis.go]: Worked with @JonathanLevi. I see this just concatenates so I understand now.

File: core/ledger/genesis/genesis.go [hunk 3]
```diff
@@ -176,14 +161,14 @@ func BuildLocal(context context.Context, spec *protos.ChaincodeSpec) (*protos.Ch
 	/*****  We will need this only when we support non-go SYSTEM chaincode ****
 	if getMode() != chaincode.DevModeUserRunsChaincode {
 		if err := core.CheckSpec(spec); err != nil {
-			genesisLogger.Debugf("check spec failed: %s", err)
+			genesisLogger.Debug("check spec failed: %s", err)
```
Comment (srderson) [inline core/ledger/genesis/genesis.go]: Remove `%s`?

File: core/ledger/genesis/genesis.go [hunk 4]
```diff
@@ -176,14 +161,14 @@ func BuildLocal(context context.Context, spec *protos.ChaincodeSpec) (*protos.Ch
...
 			return nil, err
 		}
 		// Build the spec
 		var err error
 		codePackageBytes, err = container.GetChaincodePackageBytes(spec)
 		if err != nil {
-			genesisLogger.Errorf("Error getting VM: %s", err)
+			genesisLogger.Error(fmt.Sprintf("Error getting VM: %s", err))
```
Comment (srderson) [inline core/ledger/genesis/genesis.go]: why `fmt.Sprintf` instead of `Errorf`?

File: core/ledger/genesis/genesis.go [hunk 5]
```diff
@@ -210,7 +195,7 @@ func DeployLocal(ctx context.Context, spec *protos.ChaincodeSpec, gbexists bool)
 		}
 		transaction, err = ledger.GetTransactionByUUID(chaincodeDeploymentSpec.ChaincodeSpec.ChaincodeID.Name)
 		if err != nil {
-			genesisLogger.Warningf("cannot get deployment transaction for %s - %s", chaincodeDeploymentSpec.ChaincodeSpec.ChaincodeID.Name, err)
+			genesisLogger.Warning(fmt.Sprintf("cannot get deployment transaction for %s - %s", chaincodeDeploymentSpec.ChaincodeSpec.ChaincodeID.Name, err))
```
Comment (srderson) [inline core/ledger/genesis/genesis.go]: why `fmt.Sprintf` instead of `Warningf`?

**Comment by gromeroar** [comment]
@srderson I've made the restitution of all the log calls using the master as base. the only important change made by me in this files was related to the validity period blocks removed in this PR, the log calls errors, were introduced by an error conflict in old merges.

**Comment by srderson** [comment]
LGTM. Thanks
